src/apps/lwaftr/lwaftr.lua

```diff
@@ -1037,6 +1078,11 @@ function LwAftr:from_b4(pkt)
    return self:enqueue_decapsulation(pkt, ipv4, port)
 end
 
+function LwAftr:pull ()
```

Comment:
Can we just add these to the beginning of the push function?  Seems a bit more logical, though I dunno :)